CONTRIBUTING.md

```diff
@@ -41,9 +46,7 @@ In order to contribute to the project, just do as follows:
 
 ### Improve build performance with Docker *(for Mac users only)*
 
-Please, install and run [docker-machine-nfs](https://github.com/adlogix/docker-machine-nfs).
-It will change the default file sharing of your [docker-machine](https://docs.docker.com/machine/)
-from Virtual Box Shared Folders to NFS, which is a lot faster. 
+Use [docker for mac](https://docs.docker.com/engine/installation/mac/#/docker-for-mac).
```

Comment:
I think it's still necessary, I have to run those commands if I'm running the build "outside" the container.